R/run_stress_test.R

```diff
@@ -182,7 +182,6 @@ read_and_process_and_calc <- function(args_list) {
 
   cat("-- Configuring analysis settings. \n")
 
-  flat_multiplier <- assign_flat_multiplier(asset_type = asset_type)
```

Comment:
after merging master, this will also have to be removed from `run_lrisk()`

Reply:
done